Mention that testing/development was done with Mars

datalogics-tsmith opened the pull request:
And so we can't guarantee that it will work with other versions.

Addresses [DLPJTS-63](https://jira.datalogics.com/browse/DLPJTS-63)

datalogics-tsmith committed:
Mention that testing/development was done with Mars

And so we can't guarantee that it will work
with other versions.

datalogics-devauto commented:
Build succeeded.

datalogics-devauto commented:
Build succeeded.

datalogics-kam commented on README.md:
```diff
@@ -37,6 +37,8 @@ In the project properties for **pdf-java-toolkit-samples**, select the **Maven**
 
 It will ask if you want to update the product configuration. Click **Yes**.
 
+The samples were developed and tested using [Mars](https://eclipse.org/mars/)
```
I wonder if we should mention the version. Personally, I'm up to 4.5.2 (Mars is 4.5, so this is sometimes called Mars.2).

I'm trying to find patch notes to see if there's anything that really affects us. The only way seems to be to search Bugzilla:
- [Bugs fixed in 4.5.2](https://bugs.eclipse.org/bugs/buglist.cgi?bug_status=RESOLVED&bug_status=VERIFIED&bug_status=CLOSED&classification=Eclipse&classification=RT&order=bug_id&product=Equinox&product=JDT&product=PDE&product=Platform&resolution=FIXED&target_milestone=4.5.2)
- [Bugs fixed in 4.5.1](https://bugs.eclipse.org/bugs/buglist.cgi?bug_status=RESOLVED&bug_status=VERIFIED&bug_status=CLOSED&classification=Eclipse&classification=RT&product=Equinox&product=JDT&product=PDE&product=Platform&query_format=advanced&resolution=FIXED&target_milestone=4.5.1)

In general, I try to check for updates and use the latest versions.

datalogics-tsmith replied:
I am on 4.5.0

@jrowden8, @datalogics-robl, and @datalogics-dans what versions are you guys on?

datalogics-dans replied:
@datalogics-tsmith 4.5.1 here.

datalogics-tsmith replied:
It looks like we have a few different versions in use, so does it make sense in the end to just say "Mars"? @datalogics-bhaugen @datalogics-kam

datalogics-bhaugen replied:
@datalogics-tsmith, if we said something like Mars 4.5.0, I would be happy as it seems like that is the earliest that any of us are using.

datalogics-tsmith committed:
Specify Mars 4.5.0

At the moment that seems to be the earliest that any of
the developers are using.

datalogics-tsmith commented:
done

datalogics-devauto commented:
Build succeeded.

datalogics-devauto commented:
Build succeeded.
